### PR TITLE
Update Pokemon Snap Settings

### PR DESCRIPTION
**Pull request by tony971**
GLideN64.custom.ini isn't currently working for me but these values are necessary for proper rendering.

### CODE REVIEW

**Comment by tony971** [comment]
Toggling fullscreen fixes the issues (missing textures) with copy depth buffer ro RDRAM.

File: ini/GLideN64.custom.ini [hunk 1]
```diff
@@ -308,7 +308,11 @@ frameBufferEmulation\detectCFB=1
 
 [POKEMON%20SNAP]
 Good_Name=Pokemon Snap (U)
-frameBufferEmulation\copyToRDRAM=1
```
Comment (purplemarshmallow) [inline ini/GLideN64.custom.ini]: This is needed for the pictures to show up

File: ini/GLideN64.custom.ini [hunk 2]
```diff
@@ -308,7 +308,11 @@ frameBufferEmulation\detectCFB=1
...
+generalEmulation\enableHWLighting=1
```
Comment (purplemarshmallow) [inline ini/GLideN64.custom.ini]: Why?

File: ini/GLideN64.custom.ini (hunk 1 above)
Comment (tony971) [inline ini/GLideN64.custom.ini]: It causes a flickering effect for me.

File: ini/GLideN64.custom.ini (hunk 2 above)
Comment (tony971) [inline ini/GLideN64.custom.ini]: I'll have to re-test that one when I get home. I forget what it fixed.

**Pull request closed without merging**